build.sbt

```diff
@@ -0,0 +1,47 @@
+// https://typelevel.org/sbt-typelevel/faq.html#what-is-a-base-version-anyway
+ThisBuild / tlBaseVersion := "0.0" // your current series x.y
+
+ThisBuild / organization := "zetashift"
+ThisBuild / organizationName := "Example"
+ThisBuild / startYear := Some(2023)
+ThisBuild / licenses := Seq(License.Apache2)
+ThisBuild / developers := List(
+  // your GitHub handle and name
+  tlGitHubDev("zetashift", "Rishad Sewnarain")
+)
+
+// publish to s01.oss.sonatype.org (set to true to publish to oss.sonatype.org instead)
+ThisBuild / tlSonatypeUseLegacyHost := false
+
+// publish website from this branch
+// ThisBuild / tlSitePublishBranch := Some("main")
+
+val Scala3 = "3.3.1"
+ThisBuild / crossScalaVersions := Seq(Scala3)
+ThisBuild / scalaVersion := Scala3 // the default Scala
```

Comment:
How do you feel about cross publishing to 2.12 and 2.13? That way, as well as runtime loading, we could include it in Indigo's new sbt/Mill generators to embed level definitions at build time (as we do with Aseprite.